plinth/modules/ejabberd/views.py

```diff
@@ -33,8 +37,53 @@ class EjabberdServiceView(ServiceView):
     template_name = 'ejabberd.html'
     description = ejabberd.description
     diagnostics_module_name = 'ejabberd'
+    form_class = EjabberdForm
+
+    def get_initial(self):
+        initdict = super().get_initial()
+        initdict.update({'MAM_enabled': self.is_MAM_enabled()})
+        return initdict
 
     def get_context_data(self, *args, **kwargs):
+        """Add service to the context data."""
         context = super().get_context_data(*args, **kwargs)
         context['domainname'] = ejabberd.get_domainname()
```

Comment:
@jvalleroy thanks a heap for asking whether the line above was deleted by accident; indeed it was.
Not an accident of wrong key press, but I assumed it would be taken care of the parent function, since this is inherited. But that's wrong, I checked now. Lesson learned to check all assumptions.